src/Gui/DlgSettingsDocument.ui

```diff
@@ -476,7 +476,7 @@ Common sizes are 128, 256 and 512</string>
         <item>
          <widget class="Gui::PrefSpinBox" name="prefCountBackupFiles">
           <property name="minimum">
```

Comment:
I'm a little confused. Doesn't the forum talk of a _maximum_?

Reply:
The ticket is about increasing the default minimum

Reply:
@luzpaz I have to disagree. Thread is about increasing the default value, not the minimum that user can set.
To do so, you have to add a `value` property as in https://github.com/FreeCAD/FreeCAD/blob/6ddd2a4e071b2f9732001adbdd9897bd67e5ab0d/src/Gui/DlgSettingsDocument.ui#L100
Minimum should stay unchanged.

Reply:
Oh snap, you're right. My apologies. I'm confused as to how to implement this. Can you explain?